docs/frontend-integration/sdk/sdk-methods.mdx

```diff
@@ -34,13 +34,21 @@ const { transactionHash, stSolAccountAddress } = await solidoSDK.stake({
     amount: 20,
     wallet: walletInstance,
 });
+
+// For PDA accounts
+const { transactionHash, stSolAccountAddress } = await solidoSDK.stake({
+    amount: 20,
+    wallet: walletInstance,
+    allowOwnerOffCurve: true,
+});
 ```
 
 **Parameters:**
 
 - *`amount`* - The amount of SOL-s which need to stake
 - *`wallet`* - Wallet instance from [@solana/wallet-adapter-base](https://www.npmjs.com/package/@solana/wallet-adapter-base)
 - *`setTxStage?`* - Optional callback for getting information about transaction stage
+- *`allowOwnerOffCurve?` - Allow the owner account to be a PDA (Program Derived Address), when we are creating ATA (Associated Token Account)
```

Comment:
Maybe we can add here for terms links to Solana docs to simplify users' experience?

Reply:
done